src/Generator/ValueGenerator.php

```diff
@@ -143,7 +149,7 @@ public function deleteConstant($constant)
     /**
      * Return constant list
      *
-     * @return ArrayObject
+     * @return SplArrayObject|StdlibArrayObject
```

Comment:
Would first need to dig through the framework and see where/why it was used: the initial use-case was getting rid of PHP's broken `ArrayObject` implementation (that's why the inheritance is not there).